.github/workflows/CI.yml

```diff
@@ -35,6 +35,8 @@ jobs:
         run: yarn --frozen-lockfile
       - name: Clean
         run: yarn clean
+      - name: Build
```

Comment:
Why did you add this? I'm really curious about it.

Reply:
Oh, yeah, I forgot to explain this.

The linter was failing in the CI but not locally. After looking into it, we found that a linter rule (the one that forbids you from adding two strings and suggest using interpolation instead) was throwing a false positive because the type inferred for some value was wrong. This only happened if `hardhat-core` wasn't built!

I have no idea why that happened, but the quick solution was to make sure that the build was executed before running the lint :man_shrugging: